apps/antalmanac/src/stores/Schedules.ts

```diff
@@ -1,6 +1,6 @@
-import {amber, blue, deepOrange, deepPurple, green, pink, purple,} from '@material-ui/core/colors';
+import { amber, blue, deepOrange, deepPurple, green, pink, purple } from '@material-ui/core/colors';
```

Comment:
Are these styling changes made by one of our autoformatting or linting configs? Not sure why they're popping up here when this file has been edited a couple times since we added Prettier and ESLint.